<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBBleDomSettingsPageUi</class>
 <widget class="QWidget" name="OpenRGBBleDomSettingsPageUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Test</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Use</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons">
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>saveBtn</zorder>
  <zorder>refreshBtn</zorder>
  <zorder>tableWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
